docs/user-guide/side-bar.md

```diff
@@ -1,94 +1,91 @@
-
-3D Navigation
--------------
+* **Go back** <img src="../img/button/back-extent.jpg" class="ms-docbutton"/> to the last change 
 
-Based on CesiumJS, a 3D Navigator is implemented in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) that allows you to display the map contents in 3D mode, particularly the layers will be overlaid on the globe.
+* **Go forward** <img src="../img/button/forward-extent.jpg" class="ms-docbutton"/> to the last change 
 
-* **Open** a new map then **Add** a layer.
+* **Zoom to the maximum extent** <img src="../img/button/max-extent.jpg" class="ms-docbutton"/> the map
 
-* **Click** on the 3D icon <img src="../img/button/3D-icon.jpg" class="ms-docbutton"/> in the *Sidebar*.
+3D Navigation
+-------------
 
-The scene will switch from 2D to 3D display mode and you will be guided by a brief tutorial that explain you how to interact with the map.
+Based on CesiumJS, a **3D Navigator** <img src="../img/button/3D-icon.jpg" class="ms-docbutton"/> is implemented in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) that allows the user to display the map contents in 3D mode. In particularly the layers will be overlaid on the globe and it is possible to orbit around the globe trought the compass place in the upper right corner of the map.
 
 <img src="../img/side-bar/3D-mode.jpg" class="ms-docimage"/>
 
 ## Identify Tool
 
-The Identify tool  <img src="../img/button/identify.jpg" class="ms-docbutton"/> allows you to interact with the map by clicking on features to get information stored in the attribute table of the queried feature. It is active by default as the green color suggests.
-
-* **Add** a layer to the map (e.g. USA Population).
+Getting information about the layers, which have been loaded into the map, is possible thanks to the **Identify** button <img src="../img/button/identify.jpg" class="ms-docbutton"/>, through which the attribute table, where the characteristics of the layer are stored, is queried. 
+The tool is active by default, as suggested by the green of the button. Therefore by clicking on a point on the level opens a modal window on the right side. On the window are dispalyed the requested information and the coordinates of the point.
 
-* **Click** on a feature of the layer added previously. A pop-up page will open on the right showing the requested information and the coordinates of the point.
+<img src="../img/side-bar/identify-tool.jpg" class="ms-docimage"/>
 
-<img src="../img/side-bar/get_feature_info.jpg" class="ms-docimage"/>
+The returned information are viewed in Text plain format by default. It is possible to change the format selecting the <img src="../img/button/setting_button.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu) where the user can select, from the list menu *Identify response format*, three different formats: **TEXT**, **HTML** and **PROPERTIES**.
```

Comment:
The layers information are reported in plain text by default. It is possible to change the format by selecting the <img src="../img/button/setting_button.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu) where the user can select, through the *Identify response format* menu, three different formats like: **TEXT**, **HTML** and **PROPERTIES**.